frontend/server/src/Controllers/Problem.php

```diff
@@ -2170,7 +2173,7 @@ public static function apiDetails(\OmegaUp\Request $r): array {
         } catch (\OmegaUp\Exceptions\UnauthorizedException $e) {
             // Do nothing. Not logged user can access here
         }
-        $lang = \OmegaUp\Controllers\Identity::getPreferredLanguage(
+        $lang = $r['lang'] ?? \OmegaUp\Controllers\Identity::getPreferredLanguage(
```

Comment:
```suggestion
        $lang =  \OmegaUp\Controllers\Identity::getPreferredLanguage(
```

esto no debería ser necesario porque `\OmegaUp\Controllers\Identity::getPreferredLanguage()` ya lo incluye.

igual en todos los demás lugares donde se usa este mismo patrón.

Reply:
Oh, ya vi que si está incluido en `getPreferredLanguage`, pero por algún motivo que desconozco `$_GET` no contiene el índice `lang`, intenté con `$_REQUEST`, pero tampoco obtiene nada.

Reply:
ah sabes qué podemos hacer entonces? pasarle `?\OmegaUp\Request` a `getPreferredLanguage()` y checar `$r['lang'] ?? $_GET['lang']` (o algo que a Psalm le guste). así se puede sobreescribir en las pruebas.

Reply:
Listo